tests/pending/pos/depmet_implicit_oopsla_session_2.scala

```diff
@@ -18,7 +17,7 @@ object Sessions {
 
   // friendly interface to the theory
   def runSession[S, D: Session[S]#HasDual](session: S, dual: D) =
-    ?[Session[S]#HasDual[D]].run(session, dual)
+    implicitly[Session[S]#HasDual[D]].run(session, dual)
```

Comment:
No, that's an oversight. We should keep the test as it was in pending. It worked after the first commits but then stopped working again after the final commits. Given the error message it could well be something related to existential types which dotty no longer supports, so I did not investigate further. I'll change the test to what it was an add a comm
ent.